deployment/aws-python-storage-stack-automation/storage_stack.yaml

```diff
@@ -0,0 +1,314 @@
+              - "s3:PutObject"
+              - "s3:GetObject"
+              - "s3:GetBucketTagging"
+              - "s3:ListBucket"
+              - "s3:PutEncryptionConfiguration"
+              - "s3:GetObjectTagging"
+              - "s3:ListAllMyBuckets"
+              - "s3:CreateBucket"
+              - "s3:PutObjectTagging"
+              - "sts:AssumeRole"
+              - "secretsmanager:GetSecretValue"
+              - "cloudformation:DescribeStacks"
+              - "cloudformation:GetTemplate"
+              - "cloudformation:GetTemplateSummary"
+              - "cloudformation:CreateStack"
+              - "cloudformation:ListStacks"
+              - "cloudformation:DescribeStackInstance"
+              - "cloudformation:ListStackResources"
+              - "cloudformation:DescribeStackEvents"
+              - "cloudformation:DescribeStackResource"
+              - "cloudformation:ListExports"
+              - "cloudformation:ListStackInstances"
+              - "lambda:GetLayerVersion"
+              - "lambda:PublishLayerVersion"
+              - "lambda:CreateFunction"
+              - "lambda:GetFunctionConfiguration"
+              - "lambda:DeleteFunction"
+              - "lambda:InvokeFunction"
+              - "lambda:GetEventSourceMapping"
+              - "lambda:CreateEventSourceMapping"
+              - "lambda:GetFunction"
+              - "lambda:AddPermission"
+              - "sqs:AddPermission"
+              - "sqs:GetQueueUrl"
+              - "sqs:DeleteMessage"
+              - "sqs:ReceiveMessage"
+              - "sqs:ListQueueTags"
+              - "sqs:SetQueueAttributes"
+              - "sqs:ListQueues"
+              - "sqs:GetQueueAttributes"
+              - "sqs:CreateQueue"
+              - "sqs:ListDeadLetterSourceQueues"
+              - "sqs:SendMessage"
+              - "sns:CreateTopic"
+              - "sns:Publish"
+              - "sns:SetTopicAttributes"
+              - "sns:Subscribe"
+              - "sns:GetTopicAttributes"     
+            Resource: "*"
+      Roles:
+        - !Ref lambdarole
+  storagelambda:
+    Type: AWS::Lambda::Function
+    Properties:
+      Runtime: python3.8
+      Timeout: 500
+      Handler: index.lambda_handler
+      Role: !GetAtt
+        - lambdarole
+        - Arn
+      Code:
+        ZipFile: |
+          import json
+          import logging
+          import os
+          import boto3
+          import urllib3
+          from botocore.exceptions import ClientError
+          http = urllib3.PoolManager()
+          logger = logging.getLogger()
+          logger.setLevel(logging.INFO)
+          logger.info("Logging setup Complete")
+
+          fss_key = os.environ["WS_API"]
+          queue_name = os.environ["SQS_Name"]
+          stack_name = os.environ["STACK_NAME"]
+          get_url = "https://cloudone.trendmicro.com/api/filestorage/external-id"
+          post_url = "https://cloudone.trendmicro.com/api/filestorage/stacks"
```

Comment:
How about naming it with API name, such as `stacks_api_url`. Cause I see you GET the post_url.

Reply:
updated